src/templates/layouts/layout.pug

```diff
@@ -10,6 +10,7 @@ html(lang="en")
     link(rel='stylesheet', href='https://use.fontawesome.com/releases/v5.4.2/css/all.css', integrity='sha384-/rXc/GQVaYpyDdyxK+ecHPVYJSN9bmVFBvjA/9eOB+pb3F2w2N6fc5qB9Ew5yIns', crossorigin='anonymous')
     link(href="https://fonts.googleapis.com/css?family=Nunito:400,600" rel="stylesheet")
```

Comment:
should I kill this? Not sure where it's used (I don't think it is used).

Reply:
Yeah kill this